libs/ui/inputs/src/lib/ui-inputs.module.ts

```diff
@@ -16,7 +16,6 @@ import { TextInputComponent } from './text-input/text-input.component'
 import { MatAutocompleteModule } from '@angular/material/autocomplete'
 import { MatIconModule } from '@angular/material/icon'
 import { NavigationButtonComponent } from './navigation-button/navigation-button.component'
-import { UiWidgetsModule } from '@geonetwork-ui/ui/widgets'
```

Comment:
I'm sorry but I was wrong here: the `ui-inputs` module indeed relies on the `ui-widgets` module because the autocomplete component relies on an error popup; so we cannot remove this import here